kratos/geometries/quadrilateral_2d_8.h

```diff
@@ -1539,16 +1509,20 @@ template<class TPointType> class Quadrilateral2D8
     {
```

Comment:
In this case not, because this does a loop over the integration points. For reuse the static function should be implemented for any local coordinates